pkg/tracepoint/tracepoint.go

```diff
@@ -43,6 +48,16 @@ type FieldFormat struct {
 	IsSigned bool
 }
 
+func init() {
+	// Try to use traceFS first
+	_, err := os.Stat(tracepointsPath)
+	if os.IsNotExist(err) {
+		// Fallback at using debugFS
+		usedTracepointPath = debugFsTracepointPath
+		logger.GetLogger().Warn("TraceFS not available; fallback at using debugFS", "tracefs", tracepointsPath, "debugfs", debugFsTracepointPath)
```

Comment:
Do we want a more explicit "ehy we will remove this in the near future" warning?